letsencrypt-auto

```diff
@@ -14,7 +14,12 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if ! type "sudo" > /dev/null; then
+    su -c ./$0
```

Comment:
That seems wrong to me. As I understand it, this tries to restart entire script under `su`.

The problem is, that we don't want all of the commands (e.g. virtualenv) to be run as root. Only OS dependencies bootstrapping and call to `letsencrypt` binary should be privileged.